README.md

```diff
@@ -100,14 +109,24 @@ curl -sSL --ssl https://github.com/deislabs/mystikos/releases/download/v${LATEST
 export PATH="$PATH:$(pwd)/mystikos/bin"
 ```
 
-## Install From Source
+**Debian package installation**
+```
+# change this to match the latest version
+LATEST='0.7.0'
+RELEASE="mystikos-${LATEST}-x86_64"
 
-You may also [build Mystikos from source](BUILDING.md). The build process
-will install the SGX driver and SGX-related packages for you.
+# this will create the "mystikos" directory within your current working directory
+curl -sSL --ssl https://github.com/deislabs/mystikos/releases/download/v${LATEST}/${RELEASE}.deb
+sudo dpkg -i ${RELEASE}.deb
 
-**NOTE** that Mystikos can only be built on **Ubuntu 18.04**. We are working
-on bringing Mystikos to **Ubuntu 20.04**.
+# you can use mystikos from your home directory, or any path
+export PATH="$PATH:/opt/mystikos/bin"
+```
 
+### Install From Source
+
+You may also [build Mystikos from source](BUILDING.md). The build process
```

Comment:
New docs might be needed for building from source, but those can be done after the release as well.

Reply:
I'll have to try on a clean VM, but the prereqs/Makefile should take care of the differences between 18.04 and 20.04. If not, I'll update the build docs.